site/e2e/tests/auditLogs.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { expect, test } from "@playwright/test";
+import {
+  createTemplate,
+  createWorkspace,
+  requiresEnterpriseLicense,
+} from "../helpers";
+import { beforeCoderTest } from "../hooks";
+
+test.beforeEach(async ({ page }) => await beforeCoderTest(page));
+
+test("inspecting and filtering audit logs", async ({ page }) => {
+  requiresEnterpriseLicense();
+
+  const userName = "admin";
+  // Do some stuff that should show up in the audit logs
+  const templateName = await createTemplate(page);
+  const workspaceName = await createWorkspace(page, templateName);
+
+  // Go to the audit history
+  await page.goto("/audit");
+
+  // Make sure those things we did all actually show up
+  await expect(page.getByText(`${userName} logged in`)).toBeVisible();
+  await expect(
+    page.getByText(`${userName} created template ${templateName}`),
+  ).toBeVisible();
+  await expect(
+    page.getByText(`${userName} created workspace ${workspaceName}`),
+  ).toBeVisible();
+  await expect(
+    page.getByText(`${userName} started workspace ${workspaceName}`),
+  ).toBeVisible();
+
+  // Make sure we can inspect the details of the log item
+  const createdWorkspace = page.locator(".MuiTableRow-root", {
+    hasText: `${userName} created workspace ${workspaceName}`,
+  });
```

Comment:
I don't fully understand this code, but I'm a little worried. If I'm understanding it right, the code is basically grabbing an element via the CSS class it gets from MUI, and then asserting it has the specific content?

I don't know how stable MUI class names are, but is there any way we can update this to be a role-based selector? Assuming MUI doesn't mess up their table semantics, [this element should have a `row` role as well](https://www.w3.org/WAI/ARIA/apg/practices/structural-roles/)

Reply:
I'll double check, but I didn't notice a role, and I generally prefer those.

but also MUI's classes names are a documented part of their public API, so it should be safe to depend on.